config/prod/gunicorn.conf

```diff
@@ -0,0 +1,29 @@
+# For full details, see:
+# http://docs.gunicorn.org/en/stable/settings.html
+
+# Address
+bind = '0.0.0.0:8001'
+
+# Worker configuration
+# http://docs.gunicorn.org/en/stable/design.html#async-workers
+worker_class = 'eventlet'
+
+# Rule of thumb for workers is (2 * ncpus) + 1
+# http://docs.gunicorn.org/en/stable/design.html#how-many-workers
```

Comment:
Gunicorn also supports supplying a Python file for configuration, [here's an example](https://gist.github.com/dcgoss/bfa528511114fceaa20fbfdabb35def9) from one of my projects. One benefit of this is you can dynamically compute the worker count when Gunicorn starts regardless of server, by using Python's `multiprocessing.cpu_count`